spec/e2e/asc_spec.rb

```diff
@@ -21,5 +21,37 @@
         is_expected.to eq('㈱ ｴｸｾﾙ')
       end
     end
+
+    context 'when the string contains 2-byte spaces' do
+      let!(:string) { '㈱　エクセル' } # Note the 2-byte space between characters
+
+      it 'converts 2-byte spaces to 1-byte spaces' do
+        is_expected.to eq('㈱ ｴｸｾﾙ') # Note the 1-byte space between characters
+      end
+    end
+
+    context 'when the string is empty' do
+      let!(:string) { '' }
+
+      it 'returns an empty string' do
+        is_expected.to eq('')
+      end
+    end
+
+    context 'when the string contains only 2-byte spaces' do
```

Comment:
I thought it would be good to have an English-only test like this.

```
expect: Hello, world!
actual: Hello, world!
```

Reply:
Thank you!
I will add English-only test.

Reply:
I added spec [c9157b5](https://github.com/moneyforward/xls_function/pull/18/commits/c9157b5b44afc6fd5d278e1fe657b1be8b1e17ef).